jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -497,6 +497,15 @@ def extract(self, return_bg=False, add_data=True, **kwargs):
         pix_scale_factor = self.aperture_area_along_spectral * self.spectral_cube.meta.get('PIXAR_SR', 1.0)  # noqa
         spec.meta['_pixel_scale_factor'] = pix_scale_factor
 
+        # inform the user if scale factor keyword not in metadata
+        if 'PIXAR_SR' not in self.spectral_cube.meta:
+            snackbar_message = SnackbarMessage(
+                ("PIXAR_SR FITS header keyword not found when parsing spectral cube. "
+                 "Flux/Surface Brightness will use default scale factor of 1."),
+                color="error",
```

Comment:
Should not be red unless things are crashing. People will find it alarming to see an error message and send in help calls. But I think such assumption is also dangerous enough that a simple info would not do, so let's try warning.
```suggestion
                color="warning",
```

Reply:
Will make this adjustment, I think we had discussed with Patrick the possibility that if parsed cube doesn't have the FITS header keyword, we _could/should_ allow the user to manually input the value. We also could disable the translation feature if the keyword isn't present. 

For the functional review, this is open to anyone, we get this snackbar message when the cube finishes going through the spectral extraction plugin:
`The extracted 1D spectrum was generated automatically for the entire cube. See the spectral extraction plugin for details or to perform a custom extraction.`

The message I add is somewhat hidden unless if you go to snackbar logger (since there are other messages that go to the logger after it while instantiation is finishing up).

Not that Unit Conversion is a priority over the spectral extraction message, but if there is a better way to let the user know (maybe a message in the plugin itself), this is probably better than using the snackbar.

Reply:
> but if there is a better way to let the user know (maybe a message in the plugin itself), this is probably better than using the snackbar.

I think that makes sense.  You could leave this as a warning but also put a message in the plugin if the selected cube does not have the necessary info in the metadata (as a follow-up if necessary).

Reply:
This is now added in the UI while also leaving the snackbar message as is (with a minor change in wording). Should be able to add an input field down the road if they user knows what the scale factor is for their data (and modify the traitlet that triggers this to be unicode instead of bool). 

<img width="182" alt="Screenshot 2024-07-22 at 12 51 11 PM" src="https://github.com/user-attachments/assets/f3219846-2217-4cf1-afb1-494a9edea7ac">

Reply:
great, this does the trick for now and we can always iterate on it.  Thanks!

Reply:
(I originally meant we could have a warning in the spectral extraction plugin, but maybe its best to wait to see if that is actually necessary)